### PR TITLE
Loading component

### PR DESCRIPTION
**Pull request by wensug**

### CODE REVIEW

File: src/frontend/details/details.css [hunk 1]
```diff
@@ -11,6 +11,10 @@
     justify-content: center
 }
 .details .content .cover img{
-    max-width:  60%;
+    max-width:  58%;
     border: 8px solid white;
+    background-color: rgb(245, 245, 245);
+    -webkit-box-shadow: 0 15px 20px -8px rgba(0, 0, 0, 0.2);
```
Comment (rmallols) [inline src/frontend/details/details.css]: You don't need these vendor prefixes any more, modern browsers support shadows already :)

File: src/frontend/common/loading/Loading.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import './Loading.css'
+export default function Loading(){
+    return (
+        <div className='loading'>
+            <div className='loading-loader'> </div>
+        </div>
+    )
```
Comment (rmallols) [inline src/frontend/common/loading/Loading.js]: Missing semi-colon

Comment (wensug) [reply src/frontend/common/loading/Loading.js]: I did the respective changes.

**Comment by wensug** [comment]
Ricardo agreed to the changes.

**Pull request merged**